pwiz_tools/Skyline/CommandLine.cs

```diff
@@ -906,7 +849,7 @@ private bool PerformExportOperations(CommandArgs commandArgs)
                     sharedFileName = FileEx.GetTimeStampedFileName(_skylineFile);
                 }
                 var sharedFilePath = Path.Combine(sharedFileDir, sharedFileName);
-                if (!ShareDocument(_doc, _skylineFile, sharedFilePath, commandArgs.SharedFileType, _out))
+                if (!ShareDocument(_doc, _skylineFile, sharedFilePath, commandArgs.SharedFileType, _out, commandArgs))

```

Comment:
Maybe drop commandArgs.SharedFileType as you have done elsewhere when adding commandArgs to the function arguments.

Reply:
This is one case where I have to leave that argument because in when publishing to Panorama it can be called with a different share type than the one in CommandArgs, depending on PanoramaPublishUtil.DecideShareTypeVersion.